google-iam-v1/lib/google/iam/v1/iam_policy/rest/service_stub.rb

```diff
@@ -0,0 +1,221 @@
+
+              yield result, response if block_given?
+              result
+            end
+
+            ##
+            # Baseline implementation for the test_iam_permissions REST call
+            #
+            # @param request_pb [::Google::Iam::V1::TestIamPermissionsRequest]
+            #   A request object representing the call parameters. Required.
+            # @param options [::Gapic::CallOptions]
+            #   Overrides the default settings for this call, e.g, timeout, retries etc. Optional.
+            #
+            # @yield [result, response] Access the result along with the Faraday response object
+            # @yieldparam result [::Google::Iam::V1::TestIamPermissionsResponse]
+            # @yieldparam response [::Faraday::Response]
+            #
+            # @return [::Google::Iam::V1::TestIamPermissionsResponse]
+            #   A result object deserialized from the server's reply
+            def test_iam_permissions request_pb, options = nil
+              raise ::ArgumentError, "request must be provided" if request_pb.nil?
+
+              verb, uri, query_string_params, body = ServiceStub.transcode_test_iam_permissions_request request_pb
+              query_string_params = if query_string_params.any?
+                                      query_string_params.to_h { |p| p.split("=", 2) }
+                                    else
+                                      {}
+                                    end
+
+              response = @client_stub.make_http_request(
+                verb,
+                uri:     uri,
+                body:    body || "",
+                params:  query_string_params,
+                options: options
+              )
+              result = ::Google::Iam::V1::TestIamPermissionsResponse.decode_json response.body, ignore_unknown_fields: true
+
+              yield result, response if block_given?
+              result
+            end
+
+            ##
+            # @private
+            #
+            # GRPC transcoding helper method for the set_iam_policy REST call
+            #
+            # @param request_pb [::Google::Iam::V1::SetIamPolicyRequest]
+            #   A request object representing the call parameters. Required.
+            # @return [Array(String, [String, nil], Hash{String => String})]
+            #   Uri, Body, Query string parameters
+            def self.transcode_set_iam_policy_request request_pb
+              transcoder = Gapic::Rest::GrpcTranscoder.new
+                                                      .with_bindings(
+                                                        uri_method: :post,
+                                                        uri_template: "/v1/{resource}:setIamPolicy",
+                                                        body: "*",
+                                                        matches: [
+                                                          ["resource", %r{^.*$}, true]
```

Comment:
I think rubocop will complain about the `%r{^.*$}`, wanting it to be slash-delimited instead.